cilium/datadog_checks/cilium/metrics.py

```diff
@@ -98,6 +99,36 @@
     'cilium_operator_eni_k8s_sync_queued_total': 'operator.eni.k8s_sync.queued.total',
     'cilium_operator_eni_nodes': 'operator.eni.nodes.total',
     'cilium_operator_eni_resync_total': 'operator.eni.resync.total',
+    'cilium_operator_ec2_api_duration_seconds': 'operator.ec2.api.duration.seconds',
```

Comment:
What do you think about distinguishing which metrics are only available in certain versions? We could add comments here or in the metadata.csv.

Reply:
That's a good idea, I'll add them in the metadata.csv